server/core/src/https/apidocs/mod.rs

```diff
@@ -31,6 +31,11 @@ impl Modify for SecurityAddon {
     }
 }
 
+#[derive(ToSchema)]
+/// Value is an internal implementation detail of kanidm.
+pub struct PrivateSchema {}
+
```

Comment:
Hmmm. I think changing this to actually type. But make that type Opaque might be better.